__tests__/utils.test.js

```diff
@@ -4,7 +4,7 @@ const {
   formatComments,
 } = require("../db/seeds/utils");
 
-describe("convertTimestampToDate", () => {
+describe.skip("convertTimestampToDate", () => {
```

Comment:
rather than skipping these tests (as we still need them to run and pass when we do our commits), just run `npm test <filename> `for the tests you _do_ want to run, and avoid skipping these tests manually